tests/test_dmlscheduler.py

```diff
@@ -1,7 +1,6 @@
 import pytest
```

Comment:
There are still printing statements in this file to remove.

Reply:
Removed (sorry I missed a few from debugging).